lib/set-script.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const log = require('npmlog')
+const usageUtil = require('./utils/usage.js')
+const { localPrefix } = require('./npm.js')
```

Comment:
i don't know npm's convention; is the `.js` usually included? ecosystem convention is to omit it.

Reply:
I usually omit it, but it seems that most of `npm` adds in the `.js` when using `require`:
![Screenshot from 2020-11-25 16-18-27](https://user-images.githubusercontent.com/53054099/100293872-f4b3d680-2f39-11eb-9bb8-4448e40664ee.png)